extensions/biaodian/package.json

```
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "biaodian",
+  "title": "Search Chinese Punctuation Marks",
+  "description": "Search Chinese Punctuation Marks",
+  "icon": "command-icon.png",
+  "author": "gasolin",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "biaodian",
+      "title": "Biaodian",
```

```suggestion
      "title": "Biaodian",
      "subtitle": "Chinese Punctuation Marks",
```
Sorry I forgot to add the subtitle, could you add?